client/src/components/tests/LeftAndMain-story.js

```diff
@@ -5,19 +5,32 @@ import { withKnobs, text, boolean, select } from '@storybook/addon-knobs/react';
 import { withNotes } from '@storybook/addon-notes';
 import JSXAddon from 'storybook-addon-jsx';
 import notes from '../README.md';
+import Button from 'components/Button/Button';
 
 import LeftAndMain from '../LeftAndMain';
 
-
 setAddon(JSXAddon);
 
+const topActions = [
+  { label: "Save", color: "primary", size: 'md'},
+  { label: "Delete", color: "secondary", size: 'md'},
+];
+
+const bottomActions = [
+  { label: "Edit", icon: "edit", color: "primary", size: 'md', outline: true },
+];
```

Comment:
I would discourage people from using `size` here.

```suggestion
const topActions = [
  { label: "Save", color: "primary"},
  { label: "Delete", color: "secondary"},
];

const bottomActions = [
  { label: "Edit", icon: "edit", color: "primary", outline: true },
];
```